crates/astria-bridge-withdrawer/src/bridge_withdrawer/mod.rs

```diff
@@ -45,6 +50,7 @@ use self::{
 };
 use crate::{
     api,
+    bridge_withdrawer::submitter::make_signer,
```

Comment:
If the Signer` is part of `Submitter` just provide all the values in `submitter::Builder` and let it delegate from there. Don't bubble it all the way if not needed.